src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -281,7 +282,8 @@ public List<JobModel> enumerateJobs(
       int limit,
       AuthenticatedUserRequest userReq,
       SqlSortDirection direction,
-      String className) {
+      String className,
+      Set<String> jobIdSet) {
```

Comment:
We don't use it much in TDR, but you could use `Collection<String>` for the type, as `contains()` works for all collection types, and the code only calls `contains()` on this object.

Reply:
Fair enough